README.md

```diff
@@ -90,6 +96,41 @@ To accept inbound connections, add the `--listen` flag.
 $ hsd --listen --max-inbound=20
 ```
 
+### Account and Wallet Setup
+HSD uses a hierarchical deterministic wallet structure. The hsw-cli binary is used to interact with the HSD wallet.
+
+Make sure you have a HSD node running.
+
+To view the extended public key and mnemonic details (entropy and phrase information) by:
+
+`$ hsw-cli master`
+
+To view the `default` account details, such as the `accountKey`, by:
+
+` $ hsw-cli account get default`
+
+To make a new wallet in the `default` account, use the `mkwallet` command and provide a wallet id. It is custom, but not required, to name the first wallet id `primary`.
+
+`$ hsw-cli mkwallet <WALLET_ID>`
+
+To view the wallet details such as the public key and wallet address use the `address` command and provide the wallet id and account name (`default`) previously defined:
+
+`$ hsw-cli address --id=<WALLET_ID> --account=<ACCOUNT_NAME>`
```

Comment:
This is also incorrect. The address command derives a new address. To view wallet details the `get` command can be used. For account details use `account get`.